crates/sui-framework/packages/sui-system/sources/validator_set.move

```diff
@@ -85,6 +85,21 @@ module sui_system::validator_set {
         tallying_rule_global_score: u64,
     }
 
+    /// V2 of ValidatorEpochInfoEvent containing more information about the validator.
+    struct ValidatorEpochInfoEventV2 has copy, drop {
```

Comment:
Anything else we should include here?

Reply:
Ouch, are we doing it because of the few epochs of the testnet? What would happen if we'd add a new field to the current event struct? (Are the clients going to fail to parse the `voting_power` field for these first epochs in case they'd want to consume the voting power?)